lib/strong_resources/exception_handler.rb

```diff
@@ -0,0 +1,35 @@
+module StrongResources
```

Comment:
we might want to conditionally require this file `if defined?(JsonapiErrorable)`